app/helpers/comments_helper.rb

```diff
@@ -1,23 +1,40 @@
 module CommentsHelper
   def render_comment_tree(comment)
-    content_tag :li do
-      concat content_tag(:p, comment.text)
-      concat content_tag(:div, style: "display: flex; align-items: center;") {
-        safe_concat "Posted by "
-        safe_concat link_to(comment.user.name, comment.user)
-        safe_concat "  on #{comment.created_at.strftime("%B %d, %Y at %I:%M %p")}"
-        safe_concat " - Score: "
-        safe_concat content_tag(:span, comment.score, id: "score-Comment-#{comment.id}")
-
-        safe_concat button_to("Upvote", upvote_article_comment_path(comment.article, comment), method: :post, class: "btn btn-sm upvote-button", data: {votable_type: "Comment", votable_id: comment.id, article_id: comment.article.id})
-        safe_concat button_to("Downvote", downvote_article_comment_path(comment.article, comment), method: :post, class: "btn btn-sm downvote-button", data: {votable_type: "Comment", votable_id: comment.id, article_id: comment.article.id})
+    content_tag :div do
```

Comment:
If this outer `<div>` isn't doing anything for structure or styling, I'd remove it.

Reply:
When I remove this <div>the spacing gets messed up.